packages/cli/src/commands/bundle.ts

```diff
@@ -122,8 +124,8 @@ export async function handleBundle(argv: BundleOptions, config: Config, version:
       totals.ignored += fileTotals.ignored;
 
       formatProblems(problems, {
-        format: argv.format,
-        maxProblems,
+        format: argv.format || 'codeframe',
+        maxProblems: maxProblems || 1,
```

Comment:
Why do we have 1 here and 100 [here](https://github.com/Redocly/redocly-cli/pull/1342/files#diff-6d58493339aa21bcd72df7f220f88883133ead11df05345e8fb369b77210cbeaR86)?

Reply:
I removed it because we already have default value in the `formatProblems` function